default.nix

```diff
@@ -0,0 +1,12 @@
+with import <nixpkgs> {}; {
```

Comment:
Interesting, is this the definition file for https://nixos.org/nix/?
Does cargo use it automatically or do you personally use nix package manager?

Reply:
I personally use NixOS. It tells NixOS what system libraries are needed to build the project, in this case `clang` for building bindgen which netsim uses.